runtime/cpp/CMakeLists.txt

```diff
@@ -77,6 +77,28 @@ add_compile_unit(
 
 compile_units()
 
+install(TARGETS fletcher fletcher_c fletcher_common
```

Comment:
These new install cmds are added to add a CMake config file that projects look for when trying to use fletcher.